src/main/java/gov/nist/secauto/oscal/tools/cli/core/commands/ListAllowedValuesCommand.java

```diff
@@ -261,13 +259,14 @@ private static void generateAllowedValuesList(
 
     IModuleNodeItem moduleNodeItem = INodeItemFactory.instance().newModuleNodeItem(module);
 
-    if (metapath != null) {
-      MetapathExpression filter = MetapathExpression.compile(metapath, staticContext);
-
-      DynamicContext dynamicContext = new DynamicContext(staticContext);
-      ISequence<?> sequence = filter.evaluate(moduleNodeItem, dynamicContext);
-      assert sequence != null;
-    }
+    // if (metapath != null) {
+    // MetapathExpression filter = MetapathExpression.compile(metapath,
+    // staticContext);
+    //
+    // DynamicContext dynamicContext = new DynamicContext(staticContext);
+    // ISequence<?> sequence = filter.evaluate(moduleNodeItem, dynamicContext);
+    // assert sequence != null;
+    // }
```

Comment:
Non-blocking: I see what you did there. 😉

Reply:
I started on this, but wanted to pause to get a basic command out the door.